romancal/regtest/test_wfi_flat_field.py

```diff
@@ -1,27 +1,33 @@
 import os
 import pytest
-#from asdf.commands import diff
+from asdf.commands import diff as asdf_diff
 
+from romancal.stpipe import RomanStep
 from romancal.step import FlatFieldStep
 from romancal import datamodels
 
 
-# @pytest.fixture(scope='module')
-# def run_flat_field(rtdata_module):
-#     """ Run flat field on Level 2 imaging data."""
-#
-#     rtdata = rtdata_module
-#     rtdata.get_data("WFI/image/level2.asdf")
-#     step = FlatFieldStep()
-#     step.run(rtdata.input)
+@pytest.fixture(scope='module')
+def run_flat_field(rtdata_module):
```

Comment:
I would scope all fixtures in this module to `function` and use the `rtdata` fixture instead of `rtdata_module`.  `rtdata_module` is really designed for daisy-chaining one pipeline stage to another and retaining the filenames along the way, and then making sure the data from stage 1 ends up in the same tmpdir as stage 2 pipeline is expecting to find it.

If tests in a module are independent, then just use the simpler `rtdata` fixture.